src/rpc/blockchain.cpp

```diff
@@ -2635,7 +2649,9 @@ UniValue CreateUTXOSnapshot(NodeContext& node, CChainState& chainstate, CAutoFil
     result.pushKV("coins_written", stats.coins_count);
     result.pushKV("base_hash", tip->GetBlockHash().ToString());
     result.pushKV("base_height", tip->nHeight);
-
+    result.pushKV("path", fs::PathToString(path));
```

Comment:
In commit "rpc: various fixups for dumptxoutset" (b637ffaad40ea8707bdb27334990d2bcac28cf53)

Should use `path.u8string()` here not `fs::PathToString(path)` because JSON strings (unlike log strings) need to be UTF-8